### PR TITLE
New: sampleApp - Document Product Search Result and Home components (…

### PR DESCRIPTION
**Pull request by anndiep-sfcc**
…W-7156928)

### CODE REVIEW

File: packages/storefront-lwc/src/modules/commerce/productSearchResults/reamme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!---
```
Comment (zsardone) [inline packages/storefront-lwc/src/modules/commerce/productSearchResults/reamme.md]: typo on the file name :-)

Comment (anndiep-sfcc) [reply packages/storefront-lwc/src/modules/commerce/productSearchResults/reamme.md]: Fixed. Thx!

**Comment by anndiep-sfcc** [comment]
Due to salesforce-cla issue,  closing this PR.  Will redo a new one.

**Comment by anndiep-sfcc** [comment]
Due to salesforce-cla issue,  closing this PR.  Will redo a new one.

**Pull request closed without merging**